scripts/cc-cli/src/test/integration-tests/integration.test.ts

```diff
@@ -0,0 +1,365 @@
+import { ApiPromise, BN } from "creditcoin-js";
+import { newApi } from "../../api";
+import { initKeyringPair } from "../../utils/account";
+import { signSendAndWatch } from "../../utils/tx";
+import {
+  parseAddressInternal,
+  parseAmountInternal,
+  parseHexStringInternal,
+} from "../../utils/parsing";
+import { getBalance, printBalance } from "../../utils/balance";
+import { mnemonicGenerate, mnemonicValidate } from "@polkadot/util-crypto";
+import execa from "execa";
+import { getValidatorStatus } from "../../utils/validatorStatus";
+
+function randomAccount() {
+  const seed = mnemonicGenerate();
+  const keyring = initKeyringPair(seed);
+  const address = keyring.address;
+  return { seed, keyring, address };
+}
+
+async function fundAccounts(amount: BN) {
+  const { api } = await newApi("ws://localhost:9944");
+  const stash = randomAccount();
+  const controller = randomAccount();
+  const tx = await fundAddressesFromSudo(
+    [stash.address, controller.address],
+    amount
+  );
+  await signSendAndWatch(tx, api, initKeyringPair("//Alice"));
+
+  return { stash, controller };
+}
+
+describe("integration test: validator wizard setup", () => {
+  test("new validator should appear as waiting after using the wizard setup", async () => {
+    // Fund stash and controller
+    const { stash, controller } = await fundAccounts(
+      parseAmountInternal("10000")
+    );
+    // Run wizard setup with 1k ctc ang to pair with node Bob
+    execa.commandSync(
+      `creditcoin-cli wizard --amount 1000 --url ws://localhost:9945`,
```

Comment:
Small readabilithy improvement: define these URLs as const at the top of this file, e.g. `ALICES_NODE_URL` and `BOBS_NODE_URL`